docs/content/en/index.md

```diff
@@ -89,6 +93,7 @@ This module configures [`html-validate`](https://html-validate.org/) to automati
    {
      htmlValidator: {
        usePrettier: false,
+       isVerbose: true,
```

Comment:
what about:

```suggestion
       logLevel: 'verbose', // 'verbose' | 'warning' | 'error'
```

That should give us the ability to add other levels later on if needed. And I think we could default it to 'verbose' in dev and 'warning' otherwise.

Reply:
Implemented. Tell me what you think! :heart: